numpy/core/tests/test_scalar_methods.py

```diff
@@ -90,7 +90,12 @@ def test_against_known_values(self):
             ]
         )
     ])
+
     def test_roundtrip(self, ftype, frac_vals, exp_vals):
+        if IS_MUSL and hasattr(np, 'float128') and ftype == np.float128:
+            # test is flaky on MUSLLINUX
+            return
```

Comment:
This will silently end the test. It would be better to `@pytest.mark.skipif(..., reason="flaky on some must")` the test so we have a record of this problem

Reply:
Hopefully [bc55e0f](https://github.com/numpy/numpy/pull/23069/commits/bc55e0fdf978afe19ee76dc22cb1e7510f8b427b) is better.